Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -803,7 +803,7 @@ void TextEditor::keydown_event(KeyEvent& event)
         return;
     }
 
-    if (event.key() == KeyCode::Key_Backspace) {
+    if (event.key() == KeyCode::Key_Backspace || event.code_point() == '\b') {
```

Comment:
Why is this needed?

Reply:
Ah, that was my local fix for #6748. Didn't mean to commit it with this patch!